### DISCUSSION TITLE
Help with finding icon names

### DISCUSSION THREAD

**Discussion by Ethan-Core2** [davatorium/rofi#1847] [Q&A]
Recently found out you can use icons in dmenu mode with this syntax: `echo -en "Firefox\0icon\x1ffirefox" | rofi -dmenu -no-config -show-icons`. 

However, the only one I can get to work is firefox. If I replace firefox with gimp, terminal, chromium etc, no icon shows up. I assume this is because those icons aren't in there by default. Is there a way to list all the icons you can use?

**Comment by DaveDavenport** [comment]
Rofi follows the following standard to find icons: https://specifications.freedesktop.org/icon-theme-spec/latest/

I don't directly know of an icon browser for this.
